recipes/bg-atlasapi/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "bg-atlasapi" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brainglobe/bg-atlasapi/archive/v{{ version }}.tar.gz
```

Comment:
The pypi dist was missing some files, so using the tagged github version.

Reply:
As long as there is a Github **release**, yes. A tag is not enough. I think you could also technically list both sources (GH and PyPI), extracted to different folders, but only use the GH sources for the purposes of this recipe.

Reply:
@vigji, can you confirm whether GH releases are expected for this project?

Reply:
Pinging @vigji :)

Reply:
Sorry I've missed this! Current deployment of releases should be based on v{version} tags detected in the GH Actions flow and upload to PyPI. So the GH releases are not automatic and have missed some versions. We had no strong reasons  not to automatise GH releases as well, we were just caring about PyPI and not that much about GH releases. Is this an issue?